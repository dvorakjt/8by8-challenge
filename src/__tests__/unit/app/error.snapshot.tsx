import { render, cleanup } from '@testing-library/react';
import ErrorPage from '@/app/error';

describe('ErrorPage', () => {
  afterEach(cleanup);

  it('renders the ErrorPage unchange.', () => {
    const { container } = render(<ErrorPage />);
    expect(container).toMatchSnapshot();
  });
});
