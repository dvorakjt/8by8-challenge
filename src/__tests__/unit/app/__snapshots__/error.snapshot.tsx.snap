// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage renders the ErrorPage unchange. 1`] = `
<div>
  <div
    class="outer_container"
  >
    <div
      class="inner_container_light"
    >
      <section
        class="container"
      >
        <img
          alt="Woman holding sign that says 'Oops'"
          data-nimg="1"
          decoding="async"
          height="40"
          loading="lazy"
          src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
          style="color: transparent;"
          width="40"
        />
        <h1
          class="heading"
        >
          Something
          <br />
          Went Wrong
        </h1>
      </section>
    </div>
  </div>
</div>
`;
